src/sparseml/pytorch/sparsification/quantization/modifier_quantization.py

```diff
@@ -662,7 +675,7 @@ def _enable_module_qat(self, module: Module):
         # remove qconfigs for module types in exclude_module_types
         to_exclude = []
         if self._exclude_module_types:
```

Comment:
Is this meant to be `self._exclude_module_types` or `self.exclude_module_types` also what's the difference b/w the two?

Reply:
Should be self.exclude_module_types